frontend/src/utils/webapi.js

```diff
@@ -8,10 +8,13 @@ import Node from '../Models/node.js';
 //Primitive actions supported by webapi.
 //----------------------------------------
 
+// TODO(nate): This is a hardcoded <course_id>. Change this dynamically during
+// runtime based on which course we're actually viewing
+var course_id = "42";
 var mainUrl = "";
 
 function getNode(nodeId: string){
-  let endpoint = mainUrl + `/node/${nodeId}/`;
+  let endpoint = mainUrl + '/' + course_id + `/node/${nodeId}/`;
```

Comment:
also, this is just nitpicking but matching the current camelCase would be good for consistency in variable names